base/sparse/cholmod.jl

```diff
@@ -1325,4 +1325,7 @@ function ishermitian(A::Sparse{Complex{Float64}})
     end
 end
 
+(*){Ti}(A::Symmetric{Float64,SparseMatrixCSC{Float64,Ti}}, B::SparseMatrixCSC{Float64,Ti}) = sparse(Sparse(A)*Sparse(B))
```

Comment:
last time I compared things, cholmod was quite a bit faster than pure-julia spmm, but this method is easy enough to recreate manually if you know what you're doing